### PR TITLE
pass additional props from ActionLink to a element

### PR DESCRIPTION
**Pull request by robrichard**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
   },
   "devDependencies": {
     "babel-cli": "^6.24.1",
+    "babel-plugin-transform-object-rest-spread": "^6.23.0",
```
Comment (fiddlerwoaroof) [inline package.json]: Is this necessary? Isn't this already contained in `babel-preset-stage-3`?

Comment (fiddlerwoaroof) [inline package.json]: I guess having this explicit means that it'll still work if this syntax is rejected from the standard.

Comment (robrichard) [inline package.json]: Yes you are right that it is included in `babel-preset-stage-3`. I removed it from the PR.

Comment (fiddlerwoaroof) [inline package.json]: Great, I'll merge this and add a release.  Thanks!

**Comment by fiddlerwoaroof** [comment]
Thanks for your contributions, @robrichard do you have any projects using this that you'd be willing to let us publicize as a demo of our library?

**Comment by robrichard** [comment]
Sorry - I don't, just demoing it out.

I wrote a [blog post](http://beautifulcode.1stdibs.com/2016/09/20/redux-url-router/) last year about a similar approach to routing. I'm glad to see redux state-based routing catching on.

**Comment by msumme** [comment]
Great!  Let us know if you have any questions or comments.  We have only
been using it on some internal projects so far, and welcome feedback from
the community.

**Pull request merged**